File: ios/RNIronSource.m
//
//  RNIronSource.m
//  example
//
//  Created by Loi Relia on 3/21/19.
//  Copyright © 2019 Facebook. All rights reserved.
//

#import <React/RCTBridgeModule.h>

@interface RCT_EXTERN_MODULE(RNIronSource, NSObject)
+ (BOOL)requiresMainQueueSetup
{
  return YES;
}

RCT_EXTERN_METHOD(initWithAppKey:(NSString *)appKey userId: (NSString *)userId)

@end

File: ios/RNIronSource.swift
//
//  RNIronSource.swift
//  example
//
//  Created by Loi Relia on 3/21/19.
//  Copyright © 2019 Facebook. All rights reserved.
//

import Foundation

@objc(RNIronSource)
class RNIronSource: NSObject {
  
  @objc func initWithAppKey(_ appKey: String, userId: String) {
    print("RNIronSource::initWithAppKey:: initWithAppKey", appKey,userId)
    IronSource.initWithAppKey(appKey)
    IronSource.setUserId(userId)
    ISIntegrationHelper.validateIntegration()
  }
}
